fbw-a32nx/src/systems/instruments/src/ISIS/LandingSystem.tsx

```diff
@@ -23,28 +19,27 @@ const DeviationIndicator: React.FC<DeviationIndicatorProps> = ({ deviation, avai
 
     return (
         <>
-            <rect x={0} y={0} width={200} height={20} fill="black" />
-            <circle className="StrokeWhite NoFill" cx={17} cy={10} r={3} />
-            <circle className="StrokeWhite NoFill" cx={58.5} cy={10} r={3} />
-            <circle className="StrokeWhite NoFill" cx={141.5} cy={10} r={3} />
-            <circle className="StrokeWhite NoFill" cx={183} cy={10} r={3} />
-            { available && (
-                !(aboveMaximum || belowMinimum)
-            && (// assumes 0.4 deg of deviation per dot and 41.5px distance between dots
-                <g transform={`translate(${(dots * 41.5).toFixed(5)} 0)`}>
-                    <path className="FillMagenta" d="M 84.88 10 v 0.8 l 9.45 6.7 h 11.34 l 9.45 -6.7 v -1.6 l -9.45 -6.7 h -11.34 l -9.45 6.7 z" />
+            <rect x={0} y={0} width={178.26} height={18.326} fill="black" />
+            <circle className="LSCircle NoFill" cx={14.161} cy={9.1625} r={2.9155} />
+            <circle className="LSCircle NoFill" cx={51.646} cy={9.1625} r={2.9155} />
+            <circle className="LSCircle NoFill" cx={126.616} cy={9.1625} r={2.9155} />
+            <circle className="LSCircle NoFill" cx={164.101} cy={9.1625} r={2.9155} />
+            { available && (!(aboveMaximum || belowMinimum)
+            && (// assumes 0.4 deg of deviation per dot and 37.485px distance between dots
+                <g transform={`translate(${(dots * 37.485).toFixed(5)} 0)`}>
+                    <path className="FillMagenta" d="M 75.6 9.1 v 0.8 l 8.3443 6.7 h 10.0132 l 8.3443 -6.7 v -1.6 l -8.3443 -6.7 h -10.0132 l -8.3443 6.7 z" />
                 </g>
             )
-             || belowMinimum
-            && (
-                <path className="FillMagenta" d="M 1 10 v 0.8 l 9.45 6.7 h 5.67 v-15 h -5.67 l -9.45 6.7 z" />
+            || belowMinimum
+            && (// TODO:change size and location of magenta markers
```

Comment:
Do you still plan to tackle this within this PR?

Reply:
Yeah that is an old TODO :D